app/src/main/kotlin/com/wire/android/navigation/NavigationGraph.kt

```diff
@@ -6,16 +6,28 @@ import androidx.compose.runtime.Composable
 import androidx.navigation.NavHostController
 import androidx.navigation.compose.NavHost
 import androidx.navigation.compose.composable
+import com.wire.android.ui.authentication.AuthDestination
+import com.wire.android.ui.authentication.devices.RemoveDeviceScreen
+import com.wire.android.ui.authentication.login.LoginScreen
+import com.wire.android.ui.authentication.welcome.WelcomeScreen
+import com.wire.android.ui.common.UnderConstructionScreen
+import com.wire.kalium.logic.configuration.ServerConfig
 
 @OptIn(ExperimentalMaterialApi::class)
 @ExperimentalMaterial3Api
 @Composable
 fun NavigationGraph(navController: NavHostController, startDestination: String) {
-    NavigationItem.globalNavigationItems.also { navItems ->
-        NavHost(navController, startDestination) {
-            navItems.forEach { item ->
-                composable(route = item.route, content = item.content, arguments = item.arguments)
-            }
+    NavHost(navController, startDestination) {
+        // Authentication screens
+        composable(AuthDestination.welcomeScreen) { WelcomeScreen(navController) }
```

Comment:
so why didn't you move at least `WelcomeScreen` and `RemoveDeviceScreen` into `NavigationItem`? As we agreed that this screens could be used in other places too